test/test_jit.py

```diff
@@ -19078,6 +19078,12 @@ def __init__(self, x):
                 def set_non_initialized(self, y):
                     self.bar = y  # can't assign to non-initialized attr
 
+    def test_schema_human_readable(self):
+        with self.assertRaisesRegex(RuntimeError, "nearest"):
```

Comment:
Why are we matching against "nearest" ?

Reply:
In the original issue it was the argument to mode that was printed out as octal:
mode='\156\145\141\162\145\163\164'
When it should just be 
mode='nearest'
So I am matching against nearest to make sure that we are printing it correctly

Reply:
makes sense, thanks

Reply:
I'll add in a comment in the test in a PR